src/Beta/Microsoft/Graph/Model/CloudPcRemoteActionResult.php

```diff
@@ -0,0 +1,228 @@
+*/
+namespace Beta\Microsoft\Graph\Model;
+/**
+* CloudPcRemoteActionResult class
+*
+* @category  Model
+* @package   Microsoft.Graph
+* @copyright (c) Microsoft Corporation. All rights reserved.
+* @license   https://opensource.org/licenses/MIT MIT License
+* @link      https://graph.microsoft.com
+*/
+class CloudPcRemoteActionResult extends Entity
+{
+    /**
+    * Gets the actionName
+    *
+    * @return string The actionName
+    */
+    public function getActionName()
+    {
+        if (array_key_exists("actionName", $this->_propDict)) {
+            return $this->_propDict["actionName"];
+        } else {
+            return null;
+        }
+    }
+
+    /**
+    * Sets the actionName
+    *
+    * @param string $val The value of the actionName
+    *
+    * @return CloudPcRemoteActionResult
+    */
+    public function setActionName($val)
+    {
+        $this->_propDict["actionName"] = $val;
+        return $this;
+    }
+
+    /**
+    * Gets the actionState
+    *
+    * @return ActionState The actionState
+    */
+    public function getActionState()
+    {
+        if (array_key_exists("actionState", $this->_propDict)) {
+            if (is_a($this->_propDict["actionState"], "\Beta\Microsoft\Graph\Model\ActionState")) {
+                return $this->_propDict["actionState"];
+            } else {
+                $this->_propDict["actionState"] = new ActionState($this->_propDict["actionState"]);
+                return $this->_propDict["actionState"];
+            }
+        }
+        return null;
+    }
+
+    /**
+    * Sets the actionState
+    *
+    * @param ActionState $val The value to assign to the actionState
+    *
+    * @return CloudPcRemoteActionResult The CloudPcRemoteActionResult
+    */
+    public function setActionState($val)
+    {
+        $this->_propDict["actionState"] = $val;
+         return $this;
+    }
+    /**
+    * Gets the cloudPcId
+    *
+    * @return string The cloudPcId
+    */
+    public function getCloudPcId()
+    {
+        if (array_key_exists("cloudPcId", $this->_propDict)) {
+            return $this->_propDict["cloudPcId"];
+        } else {
+            return null;
+        }
+    }
+
+    /**
+    * Sets the cloudPcId
+    *
+    * @param string $val The value of the cloudPcId
+    *
+    * @return CloudPcRemoteActionResult
+    */
+    public function setCloudPcId($val)
+    {
+        $this->_propDict["cloudPcId"] = $val;
+        return $this;
+    }
+
+    /**
+    * Gets the lastUpdatedDateTime
+    *
+    * @return \DateTime The lastUpdatedDateTime
+    */
+    public function getLastUpdatedDateTime()
+    {
+        if (array_key_exists("lastUpdatedDateTime", $this->_propDict)) {
+            if (is_a($this->_propDict["lastUpdatedDateTime"], "\DateTime")) {
+                return $this->_propDict["lastUpdatedDateTime"];
+            } else {
+                $this->_propDict["lastUpdatedDateTime"] = new \DateTime($this->_propDict["lastUpdatedDateTime"]);
```

Comment:
@SilasKenneth interesting occurrence here. We might have missed something in the Code-gen fix to add null-checks before creating DateTime objects? Not sure

Reply:
The changes I made in the codegen have not been included. Will be rerunning the pipeline to create a PR.